src/test/java/com/flightstats/hub/webhook/ActiveWebhooksIntTest.java

```diff
@@ -34,24 +33,19 @@
     private static CuratorFramework curator;
     private static SafeZooKeeperUtils zooKeeperUtils;
 
-    @BeforeClass
-    public static void setup() throws Exception {
+    @BeforeAll
+    static void setup() throws Exception {
         curator = Integration.startZooKeeper();
         zooKeeperUtils = new SafeZooKeeperUtils(curator);
     }
 
-    @Before
-    public void createWebhookLeader() throws Exception {
+    @BeforeEach
+    void createWebhookLeader() throws Exception {
         curator.create().creatingParentsIfNeeded().forPath(WEBHOOK_LEADER_PATH);
     }
 
-    @After
-    public void destroyWebhookLeaders() throws Exception {
-        curator.delete().deletingChildrenIfNeeded().forPath(WEBHOOK_LEADER_PATH);
```

Comment:
Was this unnecessary?

Reply:
It is still there, @After is now @AfterEach and it's logic is being reused in the method deletePath